simulation/src/test/resources/report.md

```diff
@@ -1,8 +1,8 @@
 # Report
 <!-- Run SimulationTest to regenerate this report. -->
 ```
-                all_nodes_500[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=79.3%	client_mean=PT6.15542S     	server_cpu=PT20M          	client_received=2000/2000	server_resps=2000	codes={200=1586, 500=414}
-                    all_nodes_500[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=73.7%	client_mean=PT3.269745S    	server_cpu=PT20M          	client_received=2000/2000	server_resps=2000	codes={200=1474, 500=526}
+                all_nodes_500[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=73.9%	client_mean=PT4.24575S     	server_cpu=PT20M          	client_received=2000/2000	server_resps=2000	codes={200=1478, 500=522}
```

Comment:
do you know what's going on with double-peak recovery graph here? client_mean seems to have improved, but overall client-perceived success rate decreased... does this mean we're burning through retries faster?

![image](https://user-images.githubusercontent.com/3473798/94922473-94189800-04ba-11eb-85e1-0bf73329fb7e.png)

Reply:
I think this is an artifact of requests sitting in per-endpoint queues being considered in-flight.